ui/src/pages/home/index.module.less

```diff
@@ -0,0 +1,28 @@
+.home {
+  :global {
+    .ant-card {
+      box-shadow: 5px 8px 15px 5px rgba(208, 216, 243, 0.6);
+      border-radius: 8px;
+    }
+  }
+
+  .cardMeta {
```

Comment:
Please use class-name pattern, which is the standard css naming convention.

Reply:
The less module needs to interact with tsx, so camelCase is used to facilitate tsx invocation.